dev/com.ibm.websphere.appserver.features/visibility/public/jsp/io.openliberty.versionless.jsp.feature

```diff
@@ -3,6 +3,6 @@ symbolicName=io.openliberty.versionless.jsp
 visibility=public
 IBM-ShortName: jsp
 Subsystem-Name: jsp
--features=io.openliberty.internal.versionless.jsp-2.2; ibm.tolerates:="2.3,3.0,3.1,4.0"
+-features=io.openliberty.internal.versionless.jsp-2.3; ibm.tolerates:="3.0,3.1,4.0"
```

Comment:
You could have left 2.2 in the list.  If the version isn't there, it will still work just like non-beta'd EE 11 features.  In a follow on PR, I suggest putting it back even without 2.2 in beta.